cli/flags.rs

```diff
@@ -350,7 +350,14 @@ Demonstrates breaking the input up by space delimiter instead of by lines:
         .long_about(
 "Automatically downloads deno_installer dependencies on first run.
 
-  deno install file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read",
+  deno install file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read
+  deno install -d /usr/local/bin file_server https://deno.land/std/http/file_server.ts --allow-net --allow-read",
```

Comment:
How about another example that has no flags:

```
deno install colors https://deno.land/std/examples/colors.ts
```

Reply:
Can you mention here that the default install directory is $HOME/.deno/bin and that people need to manually add it to their path?